packages/headless/src/features/search/legacy/search-request.ts

```diff
@@ -94,7 +94,7 @@ export const buildSearchRequest = async (
   });
 };
 
-function getFacets(state: StateNeededBySearchRequest) {
+export function getFacets(state: StateNeededBySearchRequest) {
   return sortFacets(getAllEnabledFacets(state), state.facetOrder ?? []);
 }
```

Comment:
Nope: Let's keep the legacy sarcophagus 'out-stream' 'closed' ;) (i.e. do not depend on this file outside legacy)

You can instead exfill what you need in utils and reuse it both here & where you need it.